### DISCUSSION TITLE
Loaded Model but empty response

### DISCUSSION THREAD

**Discussion by Confuzu** [oobabooga/text-generation-webui#2592] [Q&A]
I have a problem with 30  and 33B Models i  can successful load them and ask question  but the response is always empty.  

On htop  and nvtop  i see the CPU and GPU is working then this shows up  on the Terminal "Output generated in 163.87 seconds (0.08 tokens/s, 13 tokens, context 176, seed 444488510)" Both CPU and GPU  goes idle. 
But the Response goes from is typing to nothing.  even when i  wait nothing comes up.   
Im uptodate with oobabooga and  cuda 12  

I work on Linux 20.1
AMD Ryzen 9 3900X
GeForce RTX 2080 Ti 11GB
32GB RAM

`INFO:Loading guanaco-33B.ggmlv3.q4_0.bin...
INFO:llama.cpp weights detected: /media/user/HDD/Models/guanaco-33B.ggmlv3.q4_0.bin

INFO:Cache capacity is 0 bytes
llama.cpp: loading model from /media/user/HDD/Models/guanaco-33B.ggmlv3.q4_0.bin
llama_model_load_internal: format     = ggjt v3 (latest)
llama_model_load_internal: n_vocab    = 32000
llama_model_load_internal: n_ctx      = 2048
llama_model_load_internal: n_embd     = 6656
llama_model_load_internal: n_mult     = 256
llama_model_load_internal: n_head     = 52
llama_model_load_internal: n_layer    = 60
llama_model_load_internal: n_rot      = 128
llama_model_load_internal: ftype      = 2 (mostly Q4_0)
llama_model_load_internal: n_ff       = 17920
llama_model_load_internal: n_parts    = 1
llama_model_load_internal: model size = 30B
llama_model_load_internal: ggml ctx size =    0.13 MB
llama_model_load_internal: mem required  = 14589.46 MB (+ 3124.00 MB per state)
llama_model_load_internal: [cublas] offloading 18 layers to GPU
llama_model_load_internal: [cublas] total VRAM used: 5167 MB
...............................
llama_init_from_file: kv self size  = 3120.00 MB
AVX = 1 | AVX2 = 1 | AVX512 = 0 | AVX512_VBMI = 0 | AVX512_VNNI = 0 | FMA = 1 | NEON = 0 | ARM_FMA = 0 | F16C = 1 | FP16_VA = 0 | WASM_SIMD = 0 | BLAS = 1 | SSE3 = 1 | VSX = 0 | 
INFO:Replaced attention with xformers_attention
INFO:Loaded the model in 122.60 seconds.

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    91.70 ms /   200 runs   (    0.46 ms per token)
llama_print_timings: prompt eval time = 86323.86 ms /   176 tokens (  490.48 ms per token)
llama_print_timings:        eval time = 75654.74 ms /   199 runs   (  380.17 ms per token)
llama_print_timings:       total time = 163593.01 ms
Output generated in 163.87 seconds (0.08 tokens/s, 13 tokens, context 176, seed 444488510)
Llama.generate: prefix-match hit

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    87.97 ms /   200 runs   (    0.44 ms per token)
llama_print_timings: prompt eval time =  3111.70 ms /    26 tokens (  119.68 ms per token)
llama_print_timings:        eval time = 75319.02 ms /   199 runs   (  378.49 ms per token)
llama_print_timings:       total time = 86448.82 ms
Output generated in 86.72 seconds (0.15 tokens/s, 13 tokens, context 202, seed 417119352)
Llama.generate: prefix-match hit

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    90.23 ms /   200 runs   (    0.45 ms per token)
llama_print_timings: prompt eval time = 14030.05 ms /   877 tokens (   16.00 ms per token)
llama_print_timings:        eval time = 83513.26 ms /   199 runs   (  419.66 ms per token)
llama_print_timings:       total time = 98177.10 ms
Output generated in 98.47 seconds (0.13 tokens/s, 13 tokens, context 879, seed 1438135115)
Llama.generate: prefix-match hit

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    84.02 ms /   200 runs   (    0.42 ms per token)
llama_print_timings: prompt eval time =  1626.85 ms /    13 tokens (  125.14 ms per token)
llama_print_timings:        eval time = 83571.78 ms /   199 runs   (  419.96 ms per token)
llama_print_timings:       total time = 93859.99 ms
Output generated in 94.14 seconds (0.13 tokens/s, 12 tokens, context 892, seed 758505513)
Llama.generate: prefix-match hit

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    89.44 ms /   200 runs   (    0.45 ms per token)
llama_print_timings: prompt eval time = 10088.25 ms /   577 tokens (   17.48 ms per token)
llama_print_timings:        eval time = 79146.24 ms /   199 runs   (  397.72 ms per token)
llama_print_timings:       total time = 90025.04 ms
Output generated in 90.31 seconds (0.14 tokens/s, 13 tokens, context 578, seed 283211581)
Llama.generate: prefix-match hit

llama_print_timings:        load time = 86324.05 ms
llama_print_timings:      sample time =    86.02 ms /   200 runs   (    0.43 ms per token)
llama_print_timings: prompt eval time =  1623.80 ms /    13 tokens (  124.91 ms per token)
llama_print_timings:        eval time = 79583.55 ms /   199 runs   (  399.92 ms per token)
llama_print_timings:       total time = 89893.57 ms
Output generated in 90.18 seconds (0.13 tokens/s, 12 tokens, context 591, seed 1308580875)
`

**Comment by ossirytk** [comment]
I might remember thing incorrectly, but I think that the line
`INFO:llama.cpp weights detected: /media/user/HDD/Models/guanaco-33B.ggmlv3.q4_0.bin` 
Implies that this model does not include llama weights. So you either need to obtain the weights from Facebook or get a model that includes the weights.
